js/ui/status/automaticUpdates.js

```diff
@@ -286,12 +399,22 @@ var Indicator = new Lang.Class({
             return AutomaticUpdatesState.IDLE;
     },
 
-    _getActiveConnection: function() {
-        let activeConnection = this._client.get_primary_connection();
+    _alreadySentNotification: function() {
+        let connection = this._getActiveConnection();
 
-        if (!activeConnection)
-            activeConnection = this._client.get_activating_connection();
+        if (!connection)
+            return true;
```

Comment:
I'm confused... why do you return false here now? Following the same reasoning from yesterday, if you can't get an active connection, you can't get the userSetting and thus you can't tell whether you already sent a notification, so you should return false (as you were returning before), no?

Reply:
Oh! I see you replied in https://github.com/endlessm/gnome-shell/pull/245#discussion_r184092691, but is not showing up here... anyway, I get what you said about the changing semantics but still I need whatever gets returned here needs to be consistent with what's returned in the early return below...

Reply:
The semantics of the function changed. `true` now means "already notified; don't sent a notification". This, however, is just a defensive measure - if there is no active connection, this result is ignored.

Reply:
I know, I already replied to that in my previous comment (see https://github.com/endlessm/gnome-shell/pull/245/files#r184104687), and I still think this needs to be consistent with what whatever gets returned below, for `if (!userSetting)`